tox.ini

```
@@ -3,6 +3,7 @@ envlist =
     py{38,39,310}-dj32
     py{38,39,310,311}-dj41
     py{38,39,310,311}-dj42
+    py{38,39,310,311}-dj50
```

```suggestion
    py{38,39,310,311,12}-dj42
    py{310,311,12}-dj50
```
https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django